src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageManagerControl.xaml.cs

```diff
@@ -1554,7 +1581,7 @@ private void UpgradeButton_Click(object sender, RoutedEventArgs e)
             {
                 var project = Model.Context.Projects.FirstOrDefault();
                 Debug.Assert(project != null);
-                await Model.Context.UIActionEngine.UpgradeNuGetProjectAsync(Model.UIController, project);
+                await Model.Context.UIActionEngine.UpgradeNuGetProjectAsync(Model.UIController, null); // project); TODO: ActionEngine needs to take ProjectContextInfo
```

Comment:
remaining todo.
also, likely want to fix var project 2 lines above...to clarify which type of project this is refering to.